spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Another spec helper file, they wreck my head

Reply:
I think this one is a config file that you should keep from memory.